contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -0,0 +1,161 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "contrib/endpoints/src/api_manager/config_manager.h"
+#include "contrib/endpoints/src/api_manager/service_management_fetch.h"
+
+namespace google {
+namespace api_manager {
+
+namespace {
+// Default rollouts refresh interval in ms
+const int kConfigUpdateCheckInterval = 60000;
+
+const char kRolloutStrategyManaged[] = "managed";
+
+// static configs for error handling
+static std::vector<std::pair<std::string, int>> empty_configs;
```

Comment:
Renamed to kEmptyConfigs

Neither {} nor {{}} are not working.

no known conversion for argument 2 from '<brace-enclosed initializer list>' to 'std::vector<std::pair<std::basic_string<char>, int> >&'